tests/environment.py

```diff
@@ -336,3 +349,83 @@ def launch_environment_docker(unique_id: str, skip_init: bool = False, tag: str
     debug('agent2: {}'.format(agents[1].mac))
     debug('agent2 wlan0: {}'.format(agents[1].radios[0].mac))
     debug('agent2 wlan2: {}'.format(agents[1].radios[1].mac))
+
+
+class ALEntityPrplWrt(ALEntity):
+    """Abstraction of ALEntity in real device."""
+
+    def __init__(self, name: str, device: None, is_controller: bool = False):
+        self.device = device
+        self.name = name
+        self.bridge_name = 'br-lan'
+
+        if is_controller:
+            self.config_file_name = '/opt/prplmesh/config/beerocks_controller.conf'
+        else:
+            self.config_file_name = '/opt/prplmesh/config/beerocks_agent.conf'
+
+        ucc_port = self.command(("grep \"ucc_listener_port\" {} "
+                                "| cut -d'=' -f2 | cut -d\" \" -f 1").format(self.config_file_name))
```

Comment:
Personally I'd prefer to do the parsing in Python instead of shell, but that's just me I guess.
```suggestion
        ucc_port_string = self.command('grep "ucc_listener_port" {}".format(self.config_file_name))
        ucc_port, _ = ucc_port_string.split(' = ', 1)
```

Reply:
Parsing moved to Python primary. Fixed in #1358.